docs/guide/local/session-data.md

```diff
@@ -7,11 +7,13 @@ export default defineNuxtConfig({
   auth: {
     provider: {
       type: 'local',
-      sessionDataType: {
-        id: 'string | number',
-        firstName: 'string',
-        lastName: 'string'
-      }
+      session: {
+        data: {
```

Comment:
```suggestion
        dataType: {
```

https://github.com/sidebase/nuxt-auth/blob/main/src/runtime/types.ts#L204

Reply:
Ahhhh damit ... fixed it in my instance and remembered it wrong :D Thanks for catching, is fixed now :)